python/pylibcudf/pylibcudf/tests/test_scalar.py

```diff
@@ -5,11 +5,9 @@
 import pytest
 
 import pylibcudf as plc
+from pylibcudf.types import DataType, TypeId
 
-
-@pytest.fixture(scope="module")
-def np():
-    return pytest.importorskip("numpy")
+np = pytest.importorskip("numpy")
```

Comment:
I think this would skip testing the _entire_ file if numpy was not installed. That's why this was contained in a fixture

Reply:
Oh okay, will revert